### PR TITLE
Update readme to include new namespacing.

### PR DESCRIPTION
**Pull request by deric-alchemy**

### CODE REVIEW

**Comment by thebrianchen** [comment]
@deric-alchemy Just updated it to the new branch with some of the changes I made, can you resolve the merge conflicts? thanks! 

the `bc/NAMESPACES` branch will go on top of the `bc/v1.1.0` branch for the beta release

File: README.md [hunk 1]
```diff
@@ -66,25 +76,26 @@ For example, to pin to a specific version of the SDK in your `package.json` file
...
+- `getTokenBalances()`: Gets the token balances for an owner given a list of contracts.
+- `getAssetTransfers()`: Get transactions for specific addresses.
+- `getTransactionReceipts()`: Gets all transaction receipts for a given block.
 
-2. In the case that an Ethers.js method is not supported on the top-level, the Alchemy object can be used to generate an Ethers.js provider that allows access to less-common [Ethers.js methods](https://docs.ethers.io/v5/single-page/).
 
-## Accessing the full Ethers.js provider
+### Accessing the full Ethers.js provider
 
 To keep the package clean, we don't support certain uncommonly-used Ethers.js methods as top-level methods the Alchemy object - for example, `provider.formatter`. If you'd like to access these methods, simply use the `alchemy.getProvider()` function to configure the
```
Comment (thebrianchen) [inline README.md:94]: can you update this line since it should be alchemy.config.getProvider()

**Review by thebrianchen — APPROVED** [review]
thanks for fixing!

File: README.md (hunk 1 above)
Comment (deric-alchemy) [reply README.md]: done

**Pull request merged**